services/supabase/README.md

```
@@ -91,3 +91,57 @@
 ## Documentation
 
 For more detailed instructions and advanced configurations, refer to the [Supabase Local Development Guide](https://supabase.com/docs/guides/local-development).
+
+# @kindfi/supabase - Centralized Type System
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document
null

(MD025, single-title, single-h1)